cli/src/client/types.ts

```diff
@@ -1,46 +1,72 @@
 // Autogenerated from https://graphql-code-generator.com/#live-demo
 
 export type Maybe<T> = T | null;
+export type Exact<T extends { [key: string]: unknown }> = { [K in keyof T]: T[K] };
```

Comment:
I don't see this in use... Also, there is Pick<> type..

Reply:
As I wrote in the description, this file is Auto-generated, I prefer not to make changes there since they will be overridden on any future change (When it will be re-generated again)